### ISSUE TITLE
[SF][OverflowPopover] When placing buttons in the Overflow make sure their content is left-aligned

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#4452]
### **Bug Description**
According to https://wiki.wdf.sap.corp/wiki/display/visualcore/OverflowToolbar+%28Fiori+3%29+-+Popover, please make sure buttons in the overflow menu are left-aligned.
Here is the original JIRA ticket: https://jira.successfactors.com/browse/WEF-20085.

### **Expected Behavior**
https://wiki.wdf.sap.corp/wiki/display/visualcore/OverflowToolbar+%28Fiori+3%29+-+Popover#OverflowToolbar(Fiori3)Popover-LookandFeel

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--default-story
2. Shrink the screen
3. ![ui5-wc](https://user-images.githubusercontent.com/30744017/145195587-5e1ff1e2-b6f8-46db-861d-45c7f29933e2.png)

### **Context**
- UI5 Web Components version: 0.31.25
- OS/Platform: macOS
- Browser: Chrome
- Affected component: OverflowPopover

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Bussiness impact: {...}

**Comment by terezamch** [comment]
Hi @MarcusNotheis ,

The VD spec about the OverflowToolbar says that the components in the overflow must be left aligned.
In the classing UI5 this is done by overstyling the styles of the components in the overflow by the OverflowToolbar.

Could you please check if this should be done from your side or we should help somehow?

Thanks in advance,
Tereza

**Comment by MarcusNotheis** [comment]
Yes, this looks like an issue on our side, I'll pull this issue over to our repo :)
